sentinelhub/data_utils.py

```diff
@@ -99,35 +89,33 @@ def _extract_response_data(
     return df_entries
 
 
-def statistical_to_dataframe(
-    result_data: List[JsonDict], excl_stats: Optional[List[str]] = None, incl_hist: bool = False
-) -> Any:
-    """Transform (Batch) Statistical API get_data results into a pandas.DataFrame
+def statistical_to_dataframe(result_data: List[JsonDict], exclude_stats: Optional[List[str]] = None) -> Any:
+    """Transform (Batch) Statistical API results into a pandas.DataFrame
 
-    :param result_data: An input representation of (Batch) Statistical API result.
-    :param excl_stats: Unwanted statistical name.
-    :param incl_hist: Flag to transform histogram.
+    :param result_data: An input representation of (Batch) Statistical API result returned from
+        `AwsBatchResults.get_data()`. Each JsonDict in the list is a Statistical API response of an input geometry.
+    :param exclude_stats: Statistical API has `min`, `max`, `mean`, and `stDev` as default statistics.
+        The default statistics defined in this parameter will be removed from DataFrame.
```

Comment:
I am not sure this description is accurate

Reply:
Would something like `"The statistic names defined in this parameter will not be included in the DataFrame."`? This argument will be mostly used in case of the default statistics are not interested by users.

Reply:
your suggestion sounds better, yes